k8s/statefulset.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: flink-jobmanager
+spec:
+  serviceName: flink-jobmanager
+  replicas: 1
+  selector:
+    matchLabels:
+      app: flink
+      component: jobmanager
+  template:
+    metadata:
+      labels:
+        app: flink
+        component: jobmanager
+    spec:
+      securityContext:
+        fsGroup: 9999
+      containers:
+      - name: jobmanager
+        image: 281050902442.dkr.ecr.eu-central-1.amazonaws.com/hackathon-2021-stateful-flink-process/flink:1.15-SNAPSHOT
+        imagePullPolicy: Always
+        resources:
+          requests:
+            memory: "4Gi"
+            cpu: "1000m"
+          limits:
+            memory: "4Gi"
+            cpu: "1000m"
+        env:
+          - name: ENABLE_BUILT_IN_PLUGINS
+            value: "flink-s3-fs-hadoop-1.15-SNAPSHOT.jar"
+          - name: POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+        args: ["jobmanager"]
+        ports:
+        - containerPort: 6123
+          name: rpc
+        - containerPort: 6124
+          name: blob-server
+        - containerPort: 8081
+          name: webui
+        livenessProbe:
+          tcpSocket:
+            port: 6123
+          initialDelaySeconds: 30
+          periodSeconds: 60
+        securityContext:
+          runAsUser: 9999  # refers to user _flink_ from official flink image, change if necessary  
```

Comment:
Can this be merged with the `securityContext` on the spec level?

Reply:
Merged